certif/tests/unit/controllers/terms-of-service-test.js

```diff
@@ -1,11 +1,35 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
 
 module('Unit | Controller | terms-of-service', function(hooks) {
   setupTest(hooks);
+  let controller;
 
-  test('it exists', function(assert) {
-    const controller = this.owner.lookup('controller:terms-of-service');
-    assert.ok(controller);
+  module('#action submit', function(hooks) {
+
+    hooks.beforeEach(function() {
+      controller = this.owner.lookup('controller:terms-of-service');
+      controller.transitionToRoute = sinon.stub().resolves();
+      controller.currentUser = { user: { save: sinon.stub().resolves() } };
+    });
+
+    test('it should save acceptance of terms of service', async function(assert) {
+      // when
+      await controller.send('submit');
+
+      // then
+      sinon.assert.calledWith(controller.currentUser.user.save, { adapterOptions: { acceptPixCertifTermsOfService: true } });
+      assert.ok(controller);
```

Comment:
est-ce que le cette ligne a un interet?

Reply:
il y a plein de ``assert.ok`` qui me semble superflus

Reply:
@aceol Pour te répondre, je te copie/colle sans vergogne le commentaire laissé par @jbuget dans un cas identique sur PixAdmin :
` required because QUnit wants at least one expect (and does not accept Sinon's one)`
En gros, si tu n'utilises pas au moins une fois le assert de QUnit, il kiffe pas !